packages/pyright-internal/src/localization/package.nls.ru.json

```diff
@@ -370,11 +374,11 @@
         "overloadAbstractImplMismatch": "Перегрузки должны быть абстрактными, если метод абстрактный, и наоборот.",
         "overloadAbstractMismatch": "Все перегрузки должны быть абстрактными или конкретными",
         "overloadClassMethodInconsistent": "Перегрузки для \"{name}\" используют @classmethod несогласованно",
-        "overloadFinalImpl": "@final decorator should be applied only to the implementation",
-        "overloadFinalNoImpl": "Only the first overload should be marked @final",
+        "overloadFinalImpl": "Декоратор @final должен быть применён только к имплементации",
+        "overloadFinalNoImpl": "Только первая перегрузка должна быть помечена @final",
```

Comment:
note: this seems like a contradiction, but `overloadFinalImpl` is for source files (`.py`) and `overloadFinalNoImpl` is for stub files (`.pyi`)